src/coreComponents/math/TensorT/R1TensorT.h

```diff
@@ -102,6 +102,17 @@ class R1TensorT : public TensorBaseT< T_dim >
     return *this;
   }
 
+  GEOSX_HOST_DEVICE inline
+  R1TensorT & operator=( realT const (&src)[3] )
+  {
+    for( int i = 0; i < T_dim; ++i )
+    {
+      this->t_data[ i ] = src[ i ];
+    }
+
+    return *this;
```

Comment:
The signature should be `realT const (&src)[ T_dim ]`

Reply:
🤦